lib/src/main/java/net/ypresto/androidtranscoder/format/Android720pFormatStrategy.java

```diff
@@ -68,6 +76,13 @@ public MediaFormat createVideoOutputFormat(MediaFormat inputFormat) {
 
     @Override
     public MediaFormat createAudioOutputFormat(MediaFormat inputFormat) {
-        return null;
+        if (mAudioBitrate == 0 || mAudioChannels == 0) return null;
```

Comment:
Non-zero channel count (i.e. negative value) is preferred for "pass-through" because someone might want to remove existing audio channels.
Related to #23